man/org.freedesktop.machine1.xml

```diff
@@ -717,9 +724,9 @@ $ gdbus introspect --system \
       <title>Machine Objects</title>
       <para><function>CopyFromWithFlags()</function> and
       <function>CopyToWithFlags()</function> were added in version 252.</para>
-      <para><function>GetSSHInfo()</function>, <varname>VSockCID</varname>, <varname>SSHAddress</varname>
+      <para><function>GetSSHInfo()</function>, <varname>VSockCID</varname>, <varname>SSHAddress</varname>,
       and <varname>SSHPrivateKeyPath</varname> were added in version 256.</para>
-      <para><varname>LeaderPIDFDId</varname> was added in version 258.</para>
+      <para><varname>LeaderPIDFDId</varname> and <varname>Subgroup</varname> were added in version 258.</para>
```

Comment:
Please insert comma before `and`.

Reply:
i thought one would put that comma only once we add a third element to the enumeration? This one here has only two.

(note that a later commit in the same PR, will add a third anyway, where i add a comma, this discussion is kinda moot hence)